homeassistant/core.py

```diff
@@ -813,14 +829,14 @@ async def async_stop(self, exit_code: int = 0, *, force: bool = False) -> None:
         shutdown_run_callback_threadsafe(self.loop)
 
         try:
-            async with self.timeout.async_timeout(STAGE_3_SHUTDOWN_TIMEOUT):
+            async with self.timeout.async_timeout(CLOSE_STAGE_SHUTDOWN_TIMEOUT):
                 await self.async_block_till_done()
         except asyncio.TimeoutError:
             _LOGGER.warning(
-                "Timed out waiting for shutdown stage 3 to complete, the shutdown will"
+                "Timed out waiting for Home Assistant core integrations to stop, the shutdown will"
```

Comment:
Please name the message after the event so any bug reports can make it easier to figure out which event we are waiting for 
```suggestion
                "Timed out waiting for close event to be procssed, the shutdown will"
```

Also there is no need to mention Home Assistant core since this will already be logged as `homeassistant.core`